site/styles/_meta-item.scss

```diff
@@ -98,3 +98,11 @@
 .meta-item-buttons {
   margin-top: 5px;
 }
+
+// both meta detail and carousel
+.meta-item-tagline {
```

Comment:
This is overridden by `.carousel-item-caption .meta-item-content .meta-item-tagline` in the carousel and by `.meta-detail-content .meta-item-tagline` on the detail page.

Reply:
Removed